service/controller/clusterapi/v27/resource/vpccidr/create.go

```diff
@@ -4,17 +4,10 @@ import (
 	"context"
 
 	"github.com/giantswarm/microerror"
-
-	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v27/key"
 )
 
 func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
-	cluster, err := key.ToCluster(obj)
-	if err != nil {
-		return microerror.Mask(err)
-	}
-
-	err = r.addVPCCIDRToContext(ctx, cluster)
+	err := r.addVPCCIDRToContext(ctx)
```

Comment:
After the move to the new package I then noticed that the CR is not necessary anymore to add the VPC CIDR to the context.